### PR TITLE
Updated index.html

### PR DESCRIPTION
**Pull request by TheOfficialIshita**
I have updated your index.html by adding a scroll behavior and setting it to smooth. It will help to scroll the content smoothly. I hope you will like it.  :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,6 +23,11 @@
     <script src="../scripts/secrets.js"></script>
     <script src="../scripts/connector.js"></script>
     <script src="../scripts/handlers.js"></script>
+    <style>
+      *{
+        scroll-behavior: smooth;
+      }
+    </style>
```
Comment (dev-prakhar) [inline index.html]: Let's create a new folder `styles` and add a `.css` file there. Let's not add inline CSS here.
Also, can you explain a bit more, why is this necessary?

**Review by dev-prakhar — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (TheOfficialIshita) [reply index.html]: Ya... definitely it's a good practice to add a new .css file and write the code there. 
The scroll-behavior property in CSS allows us to define whether the scroll location of the browser jumps to a new location or smoothly animates the transition when a user clicks a link that targets an anchored position within a scrolling box.
For Example, In your Code, when you click:- Play with Notification link, you will scroll down smoothly instead of directly jumping to it.
You can learn more about it from:- https://www.w3schools.com/cssref/pr_scroll-behavior.asp
👍

File: index.html [hunk 2]
```diff
@@ -23,6 +23,7 @@
     <script src="../scripts/secrets.js"></script>
     <script src="../scripts/connector.js"></script>
     <script src="../scripts/handlers.js"></script>
+    <link rel="stylesheet" href="styles/style.css">
```
Comment (dev-prakhar) [inline index.html]: These are the script tags. Please move the css with the css tags
https://github.com/dev-prakhar/push-me-out-client/blob/main/index.html#L15

**Review by dev-prakhar — APPROVED** [review]
